lib/geolocation.js

```diff
@@ -70,10 +72,14 @@ async function setLocationWithIdb (idb, latitude, longitude) {
  * @throws {Error} If it failed to set the location
  */
 async function setLocationWithAppleScript (sim, latitude, longitude, menu = 'Debug') {
-  const decimalSeparator = (0.1).toLocaleString().replace(/\d/g, '');
   // Make sure system-wide decimal separator is used
-  const latitudeStr = `${latitude}`.replace(/\D/g, decimalSeparator);
-  const longitudeStr = `${longitude}`.replace(/\D/g, decimalSeparator);
+  const decimalSeparator = (await sim.executeUIClientScript(`
+    use framework "Foundation"
+    return (decimalSeparator of init() of alloc() of NSNumberFormatter of current application as string)
+  `)).trim();
+
+  const [latitudeStr, longitudeStr] = [latitude, longitude].map(s => s.replace(/\D/g, decimalSeparator))
```

Comment:
Please use the following syntax for lambdas: `(s) => s.replace(/\D/g, decimalSeparator)`

Reply:
s.replace will fail if latitude or longitude is a number

Reply:
good catch, linted and fixed